src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -129,6 +130,19 @@ SemiclassicalJacobi{T}(t, a, b, c) where T = SemiclassicalJacobi(convert(T,t), c
 
 WeightedSemiclassicalJacobi{T}(t, a, b, c, P...) where T = SemiclassicalJacobiWeight{T}(convert(T,t), convert(T,a), convert(T,b), convert(T,c)) .* SemiclassicalJacobi{T}(convert(T,t), convert(T,a), convert(T,b), convert(T,c), P...)
 
+# Returns α, β such that P1(x) = β(x-α)
+function _linear_coefficients(t, a, b, c)
+    Γ = let t=t,b=b,c=c
```

Comment:
I think it's best to avoid closures/anonymous functions since its harder to reason about in terms of compile penality...

Reply:
Changed.